src/pybind/mgr/dashboard/frontend/src/app/shared/components/select-badges/select-badges.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PopoverModule } from 'ngx-bootstrap';
+
+import { configureTestBed } from '../../unit-test-helper';
+import { SelectBadgesComponent } from './select-badges.component';
+
+describe('SelectBadgesComponent', () => {
+  let component: SelectBadgesComponent;
+  let fixture: ComponentFixture<SelectBadgesComponent>;
+
+  configureTestBed({
+    declarations: [SelectBadgesComponent],
+    imports: [PopoverModule.forRoot()]
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SelectBadgesComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Please add tests for every functionality of the shared component.

Reply:
@Devp00l I've added tests to the `cd-select-badges` component.